src/mavoscript.js

```diff
@@ -64,7 +80,7 @@ var _ = Mavo.Script = {
 			});
 		}
 
-		o.identity = o.identity === undefined? 0 : o.identity;
+		o.default = o.default === undefined? 0 : o.default;
```

Comment:
I would expect a separate PR for the renaming. In general, please try to keep PRs small and atomic, it's much easier to review them which means they get merged faster.